README.md

```diff
@@ -486,7 +486,8 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Balaji Sigamani](https://github.com/balajisa09)
   - [Balikis Oyeleye](https://github.com/Qreamville)
   - [Baraa Baba](https://github.com/baraa-baba)
-  - [Barkatul Mujauddin](https://github.com/barkatul)
+  - [Barlapally Krishna Reddy] (https://github.com/krishnareddy067)
```

Comment:
```suggestion
  - [Barlapally Krishna Reddy](https://github.com/krishnareddy067)
```

Also, this needs to be moved so that it's between Barkatul Mujauddin and Barphie